vars/cacheStore.groovy

```diff
@@ -16,11 +16,13 @@ void call(String s3Bucket, ArrayList<String> cacheKeys, ArrayList<String> itemsT
 
   echo 'Pushing cache to AWS'
   cacheKeys.each { cacheKey ->
+    String serializedCacheKey = cacheKey.replaceAll("\\W", "")
```

Comment:
Is there a way to extract this into a `serialize` function that we can use both here and in `cacheRestore`? This way we can keep them aligned and have a new helper function if we need serialization elsewhere

Reply:
done